jsonschema-core.xml

```diff
@@ -792,47 +798,43 @@
         <section title="Meta-Schemas and Vocabularies" anchor="vocabulary">
             <t>
                 Two concepts, meta-schemas and vocabularies, are used to inform an implementation
-                how to interpret a schema.  A schema S declares its meta-schema M with the "$schema"
-                keyword, and meta-schemas declare vocabularies with the "$vocabulary" keyword.
-                The vocabularies declared in M are those that are expected to be used in S.
-                The meta-schema M may itself use a different set of vocabularies, which are
-                declared in its own meta-schema, M'.
-            </t>
-            <t>
-                The role of the meta-schema is to constrain the structure of conforming schemas,
-                as well as simplify the process of composing multiple vocabularies into a usable
-                feature set.  Schema authoring is expected to be a common activity, so
-                schema authors need only understand how to reference a single meta-schema.
-            </t>
-            <t>
-                Meta-schema authoring is an advanced usage of JSON Schema, so the design of
-                meta-schema features emphasizes flexibility over simplicity.
+                how to interpret a schema.  Every schema has a meta-schema, typically identified
```

Comment:
> Every schema has a meta-schema.

Later you say "If absent from the root schema, the resulting behavior is implementation-defined."  I realize they don't _technically_ conflict, but it might be nice to have something to the effect of "even if it's not explicitly defined by the `$schema` keyword" rather that the casual "typically defined by..."